### ISSUE TITLE
fetch errors

### ISSUE THREAD

**Issue by murqacher** [StefanBossbaly/CopperheadOS-Docker#1]
```
sudo docker run \
    -v /srv/src:/srv/src \
    -v /srv/chromium:/srv/chromium \
    -v /srv/ccache:/srv/ccache \
    -v /srv/keys/:/srv/keys \
    -v /srv/tmp:/srv/tmp \
    -v /srv/zips:/srv/zips \
    -e USE_CCACHE=1 \
    -e CCACHE_SIZE="0" \
    -e SIGNATURE_SPOOFING="no" \
    -e DEVICE="sailfish" \
    -e BUILD_TAG="OPM2.171019.029.2018.04.19.04" \
    -e BUILD_ID="OPM2.171019.029" \
    -e NUM_OF_THREADS=20 \
    bflux/copperheados-docker
```
exited with error: Exited sync due to fetch errors

when I run again I get

```
fatal: unable to access 'https://bitbucket.org/copperhead-security/platform_external_chromium/': GnuTLS recv error (-110): The TLS connection was non-properly terminated.
error: Cannot fetch platform_external_chromium
```

**Comment by StefanBossbaly** [comment]
This is a [known issue](https://www.reddit.com/r/CopperheadOS/comments/8e9m42/no_longer_going_to_be_distributing_chromium/) and isn't necessarily a problem with the build container. They no longer support having chromium prebuilt and you have to build it from source (which this container supports). Unfortunately they removed the bitbucket project and have not update the manifest until this [commit](https://github.com/CopperheadOS/platform_manifest/commit/6b0105d19a01669bac69bf4c74866b39b6eba40f) which is not included in the  `OPM2.171019.029.2018.04.19.04` tag. The workaround is to change the `remote` from `bitbucket` to `github` and then run the `repo sync`. You can launch the container and do this manually by running

```
sudo docker run -it \
    -v /srv/src:/srv/src \
    -v /srv/chromium:/srv/chromium \
    -v /srv/ccache:/srv/ccache \
    -v /srv/keys/:/srv/keys \
    -v /srv/tmp:/srv/tmp \
    -v /srv/zips:/srv/zips \
    -e USE_CCACHE=1 \
    -e CCACHE_SIZE="0" \
    -e SIGNATURE_SPOOFING="no" \
    -e DEVICE="sailfish" \
    -e BUILD_TAG="OPM2.171019.029.2018.04.19.04" \
    -e BUILD_ID="OPM2.171019.029" \
    -e NUM_OF_THREADS=20 \
    --entrypoint /bin/bash \
    bflux/copperheados-docker
```

Let me know if you need any help. I was able to get a build working using this method. This will be fixed in the next tag which is why I haven't included a workaround in the container.

**Comment by murqacher** [comment]
```
sudo docker run \
    -v /srv/src:/srv/src \
    -v /srv/chromium:/srv/chromium \
    -v /srv/ccache:/srv/ccache \
    -v /srv/keys/:/srv/keys \
    -v /srv/tmp:/srv/tmp \
    -v /srv/zips:/srv/zips \
    -e USE_CCACHE=1 \
    -e CCACHE_SIZE="0" \
    -e SIGNATURE_SPOOFING="no" \
    -e DEVICE="sailfish" \
    -e BUILD_TAG="OPM4.171019.016.B1.2018.05.08.01" \
    -e BUILD_ID="OPM4.171019.016.B1" \
    -e CHROMIUM_RELEASE_NAME="66.0.3359.126" \
    -e CHROMIUM_RELEASE_CODE="335912660" \
    -e NUM_OF_THREADS=20 \
    bflux/copperheados-docker
```
fatal: could not open '/srv/chromium/chromium_patches/*.patch' for reading: No such file or directory

fixed with deleting quotes here: https://github.com/StefanBossbaly/CopperheadOS-Docker/blob/c2418b74da98dddcf5e37b0c359f22da6a9a846b/src/docker_entrypoint.sh#L98

**Comment by StefanBossbaly** [comment]
Thanks for the catch. Will update in an upcoming commit.

**Comment by StefanBossbaly** [comment]
Closed via fda5dfa411e7fe3ca98dca22304a37fb07392447
